src/PerfView/PerfViewData.cs

```diff
@@ -9144,6 +9171,36 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
                         stackSource.DoneAddingSamples();
                         return stackSource;
                     }
+                case "Contention":
+                    {
```

Comment:
It is safe to add both of these views for ETW traces as well.  Would you mind adding these to the `Advanced` group there as well?  You can also remove the setting for `OnlyManagedCodeStacks` when adding for ETW traces, since they can capture both native and managed frames.

Reply:
Added them but I didn't test that it works

Reply:
Yes, that's right.

Reply:
I did the testing, it works! However, I had a lot of trouble collecting stacks on Windows with PerfView (it works fine with `dotnet-trace` for the same process). I used this command line at the end, it does collect the stacks but for some reason the symbols are not resolved correctly; I get `?!?`:

```
PerfView.exe  "/DataFile:PerfViewData.etl" /BufferSizeMB:256 /StackCompression /CircularMB:500 /KernelEvents:None /ClrEvents:None /Providers:"Microsoft-Windows-DotNETRuntime:0x40000004000:Verbose:@StacksEnabled=true;@ProcessIDFilter=1234" /NoGui /NoNGenRundown collect
```

Reply:
Glad to hear that it worked.  The reason stacks didn't work is because you're missing kernel and CLR events.  If you remove `/KernelEvents:None` and `/ClrEvents:None` it should work.

Reply:
I tried with your suggestions and indeed it fixed the symbols however it now collects a lot of other events (like CPU samples) and it's system-wide (guess because my process filter only worked for dotnet events and not kernel ones?)

Reply:
Yes, that's expected.  The filter will only impact the provider that specifies it, and I don't believe there is a way to do so for the kernel provider.